.github/workflows/test-application.yaml

```diff
@@ -65,15 +65,7 @@ jobs:
               uses: shivammathur/setup-php@v2
               with:
                   php-version: ${{ matrix.php-version }}
-                  tools: 'composer:v2'
-
-            - name: Globally install symfony/flex
-              if: matrix.symfony-require != ''
-              run: composer global require --no-progress --no-scripts --no-plugins symfony/flex
-
-            - name: Remove dev dependencies not compatible with Symfony 6
-              if: matrix.symfony-require == '6.0.*'
-              run: composer remove liip/rmt --dev --no-update --no-interaction
+                  tools: 'composer:v2, flex'
```

Comment:
should we have the `,` or not? here you have one, below you don't

Reply:
Good catch, was copy paste error. It should indeed be seperated by a comma! Fixed!

Reply:
yep, previously it only installed flex for dbal